src/lib/SkyraClient.ts

```diff
@@ -7,6 +7,7 @@ import { Client as VezaClient } from 'veza';
 import { Webhook } from 'discord.js';
 import { FSWatcher } from 'chokidar';
 import { DashboardClient } from 'klasa-dashboard-hooks';
+import { InfluxDB, WriteApi, WritePrecision } from '@influxdata/influxdb-client';
```

Comment:
```suggestion
import { InfluxDB, WriteApi, WritePrecision } from '@influxdata/influxdb-client';
import { Webhook } from 'discord.js';
import { FSWatcher } from 'chokidar';
import { DashboardClient } from 'klasa-dashboard-hooks';
```

This is basically what organize imports would make of it.